components/script/dom/bindings/reflector.rs

```diff
@@ -43,6 +43,16 @@ where
 }
 
 pub(crate) trait DomGlobalGeneric<D: DomTypes>: DomObject {
+    /// Returns the [`GlobalScope`] of the realm that the [`DomObject`] was created in.  If this
+    /// object is a `Node`, this will be different from it's owning `Document` if adopted by. For
```

Comment:
"if adopted by" seems like a strange ending.

Reply:
I just duplicated the existing doc comment for the method.